app/views/CheckYourAnswersView.scala.html

```diff
@@ -38,4 +38,13 @@
             }
         }
     }
+
```

Comment:
@somaraja could we remove this as it won't be needed. The submit button will call the onSubmit method in the controller which I'm implementing on another branch.